examples/Makefile

```diff
@@ -11,7 +11,8 @@ install: install-observability install-argo
 	@echo "-----------------------------------"
 	helm repo add klt https://charts.lifecycle.keptn.sh
 	helm repo update
-	helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
+	helm upgrade --install keptn klt/klt -n $(TOOLKIT_NAMESPACE) --create-namespace --wait
```

Comment:
I am using this for our integration tests, what do you think of using it here too? 
	KLT_NAMESPACE=$(shell kubectl get pod -l  control-plane=lifecycle-operator -A  -o=jsonpath='{.items[0].metadata.namespace}')